jsonnet/thanos/thanos.jsonnet

```diff
@@ -0,0 +1,15 @@
+local po = (import 'config.libsonnet').thanos;
+
+{
+  'prometheus.yaml': po.prometheus,
+  'prometheus-cluster-role.yaml': po.clusterRole,
+  'prometheus-cluster-role-binding.yaml': po.clusterRoleBinding,
+  'prometheus-service.yaml': po.service,
+  'prometheus-servicemonitor.yaml': po.serviceMonitor,
+  'query-deployment.yaml': po.queryDeployment,
+  'query-service.yaml': po.queryService,
+  'sidecar-service.yaml': po.sidecarService,
+  'thanos-ruler.yaml': po.thanosRuler,
+  'prometheus-rule.yaml': po.prometheusRule,
+  'thanos-ruler-service.yaml': po.thanosRulerService,
+}
```

Comment:
Since this is not meant to be used as a library, we should move it to `scripts/generate/`. Same way as we have `prometheus-operator-rbac.jsonnet`